hotel-reservation-system/src/components/Hotel/ServiceTable.js

```diff
@@ -182,50 +250,56 @@ export default function ServiceTable({ hotelId }) {
         <TablePagination
           rowsPerPageOptions={[5, 10, 50]}
           component="div"
-          count={maxNumberOfServices}
+          count={hotelId === undefined ? services.length : maxNumberOfServices}
```

Comment:
It would be better to check positive conditions. Like hotelId ? maxNumberOfServices : services.length. But if it really matters hotelId to be undefined but not null then leave this condition as it is.

Reply:
replaced by positive conditions in every class